openadapt/config.py

```diff
@@ -396,9 +401,10 @@ def maybe_obfuscate(key: str, val: Any) -> Any:
     Returns:
         str: The possibly obfuscated configuration parameter value.
     """
-    OBFUSCATE_KEY_PARTS = ("KEY", "PASSWORD", "TOKEN")
+    OBFUSCATE_KEY_PARTS = ("KEY", "PASSWORD", "TOKEN", "AUTHKEY", "PORT", "ADDRESS")
```

Comment:
I don't think we need to obfuscate these do we?

Reply:
Done here: a73101dc752c0a24542d0f119211571ec719a147